ra/ra.go

```diff
@@ -683,12 +684,12 @@ func (ra *RegistrationAuthorityImpl) checkAuthorizations(ctx context.Context, na
 		} else if authz.Expires.Before(now) {
 			badNames = append(badNames, name)
 		} else if authz.Expires.Before(caaRecheckTime) {
-			recheckNames = append(recheckNames, name)
+			recheckAuths = append(recheckAuths, *authz)
```

Comment:
I _think_ it might be preferable to make `recheckCAA` accept `[]*core.Authorization` instead of dropping the pointer here. Thoughts?

Reply:
I decided to drop the pointer there because there it is already checked to be not `nil`. Then I don't need additional `nil` checks in `recheckCAA`. I don't think it's useful to allow to pass `nil` elements to `recheckCAA`. What are your reasons for `recheckCAA` to accept `[]*core.Authorization`?

Reply:
I was thinking that it would save copying the Authorizations and associated Challenges but I don't think it's a super compelling reason.

Given that the existing function doesn't take a pointer type and as you mentioned it would require an additional nil check I'm happy to see it left as-is. Thanks for pushing back with an explanation :-)